rubocop/.gitignore

```diff
@@ -0,0 +1,12 @@
+/.bundle/
+/.yardoc
+/Gemfile.lock
```

Comment:
It is a standard behaviour for all gems. gemspec should strictly describe with wich gems it works.
In this case, both runtime dependencies are locked to exact versions, so there is no real reason to ignore it, but it is still a common behaviour so i don't think we need to change it